actors/uploader/src/validators/smartyValidator.js

```diff
@@ -0,0 +1,18 @@
+export function SmartyValidator(item) {
```

Comment:
není to konstruktor, proč používá naming convention pro konstruktory?

Reply:
tpyo, vyreseno smazanim 🔥